src/main/java/com/redhat/red/build/koji/model/xmlrpc/KojiArchiveInfo.java

```diff
@@ -148,6 +168,46 @@ public void setClassifier( String classifier )
         this.classifier = classifier;
     }
 
+    public String getRelPath()
```

Comment:
It would be great if we could either rename this, or else provide an alias getter/setter that uses a name the caller can understand more easily. The term 'relpath' is jargonized for koji experts...I've used koji lightly for 7 years, and I have no idea what this is.